sdk/src/main/scala/com/horizen/block/SidechainBlock.scala

```diff
@@ -213,6 +213,7 @@ object SidechainBlock extends ScorexEncoding {
   val MAX_SIDECHAIN_TXS_NUMBER: Int = 1000
   val ModifierTypeId: ModifierTypeId = scorex.core.ModifierTypeId @@ 3.toByte
   val BLOCK_VERSION: Block.Version = 1: Byte
+  val BlockIdLenght = 64
```

Comment:
BlockIdLeng**ht** -> BlockIdLeng**th**

Moreover, it is the length of the id in a string hex form